invenio_app_ils/config.py

```diff
@@ -1092,6 +1087,9 @@ def _(x):
 ILS_INDEXER_TASK_DELAY = timedelta(seconds=5)
 """Time delay that indexers spawning their asynchronous celery tasks."""
 
+ILS_ACCESS_LIST_SEARCH = access_list_search
```

Comment:
In general in Invenio I saw this called as `factory`, I would maybe stick to this convention.
Let's discuss about naming face to face